### PR TITLE
Create a didcomm pickup client javascript client component

### LINKED ISSUE #210: Create a didcomm pickup client javascript client component.
Branch Link: https://github.com/adorsys/didcomm-messaging-clients-utilities/tree/Create-a-didcomm-pickup-client-javascript-client-component

### PR DESCRIPTION
**Pull request by Tekum-Emmanuella**
PR introduces functions for constructing and securely sending DIDComm pickup requests, delivery requests, and message receipts. It includes encryption, message packing, and unpacking to enhance secure communication for a pickup-client javascript-client

### CODE REVIEW

**Review by Blindspot22 — APPROVED** [review]
LGTM :rocket:

**Review by Christiantyemele — CHANGES_REQUESTED** [review]
Please go through the reviews here they are also repeating in your ticket

File: libs/message-handler/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    mockAdapter.restore();
+  });
+
+  it('should send a pickup request', async () => {
+    const mockResponse = { success: true };
+    mockAdapter
+      .onPost('https://example.com/mediation-endpoint')
+      .reply(200, mockResponse);
+  });
+});
```
Comment (Christiantyemele) [inline libs/message-handler/src/index.ts]: What is this file for

File: libs/message-handler/src/pickup.test.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    const packmsg = await pack_encrypt(msg, FROM);
+    console.log(packmsg); // Log the packed message for debugging
+
+    // Unpack and handle the response
+    const unpacksg = await handleResponse(FROM, packmsg as string);
+    console.log(unpacksg); // Log the unpacked message for debugging
+
+    // Expect the packed message to not be null
+    expect(packmsg).not.toBeNull();
+  });
```
Comment (Christiantyemele) [inline libs/message-handler/src/pickup.test.ts]: Remove the console logs here use expects instead

File: libs/message-handler/tsconfig.node.json [hunk 3]
```diff
@@ -18,5 +18,5 @@
     "noUnusedParameters": true,
     "noFallthroughCasesInSwitch": true
   },
-  "include": ["src"]
+  "include": ["src", "src/pickup.test.ts"]
 }
```
Comment (Christiantyemele) [inline libs/message-handler/tsconfig.node.json]: including the src directory is enough here 
```suggestion
  "include": ["src"]
}
```

**Review by Christiantyemele — CHANGES_REQUESTED** [review]

**Review by chendiblessing — APPROVED** [review]

**Review by Christiantyemele — APPROVED** [review]

**Pull request merged**